api/handler/put.go

```diff
@@ -762,6 +747,27 @@ func (h *handler) CreateBucketHandler(w http.ResponseWriter, r *http.Request) {
 	api.WriteSuccessResponseHeadersOnly(w)
 }
 
+func (h handler) setPolicy(prm *layer.CreateBucketParams, locationConstraint string, userPolicies []*accessbox.ContainerPolicy) {
+	prm.Policy = h.cfg.Policy.Default
+
+	if locationConstraint == "" {
+		return
+	}
+
+	if policy, ok := h.cfg.Policy.RegionMap[locationConstraint]; ok {
```

Comment:
So it is guaranteed that `h.cfg.Policy.RegionMap` map contains at least default policy, right?

Reply:
Not. It contains only policies from `placement_policy.region_mapping` config value